server.js

```diff
@@ -1,10 +1,71 @@
-const express = require('express')
-const app = express()
-const port = 3001
-app.get('/ping' , (req, res) => {
-    res.send('Hello everyone')
-})
-
-app.listen(port , () => {
-    console.log('server is running');
-})
\ No newline at end of file
+// const express = require('express')
+// const app = express()
+// const port = 7000
+// const mongoose = require('mongoose')
+
+// const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority';
+
+// mongoose.connect(uri)
+// .then(() => {
+//     app.get('/', (req, res) => {
+//         res.json({connection: 'Connected'})
+//     })
+// })
+// .catch(() =>{
+//     app.get('/' ,(req , res) => {
+//         res.json({connection: 'Disconnected'})
+//     })
+// })
+
+// // app.get('/',(req, res)=>{
+// //     console.log(req.url);
+// //     mongoose.connect(uri)
+// //     .then(() => {
+// //         res.send("connected to mongo successfully");
+// //     }).catch((error)=>{
+// //         res.send("got error while connecting", error);
+// //     })
+// // })
+
+
+// app.listen(port, () => {
+//     console.log("server is up and runnig at port:",port);
+// })
+
+const express = require('express');
+
+const { startDatabase, stopDatabase, isConnected } = require('./db');
+const app = express();
+
+const port = process.env.PUBLIC_PORT ?? 8000;
+
+app.get('/', (req, res) => {
```

Comment:
Consider adding a health check endpoint to your server. This endpoint can be used to check the status of your server and database connection. [medium]